hugo/assets/scripts/statemachine.tsx

```diff
@@ -0,0 +1,356 @@
+import {
```

Comment:
This file is very long, you should split it somehow to smaller chucks. Maybe one per concept/class